### PR TITLE
Create 2023-07-06

### PR DESCRIPTION
**Pull request by robcraig-LF**
Adds meeting agenda/minutes SPDX Serialization Team Meeting 2023-07-06

### CODE REVIEW

File: serialisation/2023-07-06 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - mention that context will be versioned
+    - e.g. http://spdx.org/rdf/v3/context.json  vs   http://spdx.org/rdf/v3.1/context.json
+
+- Serialization formats
+  - we have said that we want to support multiple serialization formats
+  - RDF definitely
+  - JSON can be done in multiple ways
+    - JSON-LD-as-JSON
+    - simple JSON
+  - other formats, possibly including a very simple printf-generated
```
Comment (goneall) [inline serialisation/2023-07-06:33]: I would like to add the following which was discussed in the meeting.

@zvr - let me know if you agree:

- Next meeting, we'll focus on a decision of the JSON-LD format can be a more general JSON format or if we need a second JSON format
- If we decide we need a second format, we will document the issues with the JSON-LD
- If we decide we need a second format, we will review David's proposal and decide if it meets the needs for a second JSON format or document the issues preventing it from being accepted

**Review by goneall — CHANGES_REQUESTED** [review]
Suggest adding the comments only if @zvr agrees

File: serialisation/2023-07-06 (hunk 1 above)
Comment (zvr) [reply serialisation/2023-07-06:33]: Completly agree -- I did not type all this, but it was definitely in our call

**Comment by goneall** [comment]
@robcraig-LF - I went ahead and added the additional bullets to the commit.

I also noticed that the filename was missing the .md file extension.

I'll create a commit to rename the file - you may want to check the other files to make sure they end in .md.

**Review by goneall — APPROVED** [review]

**Pull request merged**